src/ProfileView.jl

```diff
@@ -33,32 +33,34 @@ const gccolor = colorant"red"
 const colors = distinguishable_colors(13, [bkg,fontcolor,gccolor],
                                       lchoices=Float64[65, 70, 75, 80],
                                       cchoices=Float64[0, 50, 60, 70],
-                                      hchoices=linspace(0, 330, 24))[4:end]
+                                      hchoices=range(0, stop=330, length=24))[4:end]
 
 function have_display()
-    !is_unix() && return true
-    is_apple() && return true
+    !Sys.isunix() && return true
+    Sys.isapple() && return true
     return haskey(ENV, "DISPLAY")
 end
 
+include("ProfileViewSVG.jl")
+include("ProfileViewGtk.jl")
```

Comment:
So, these used to be loaded conditionally. The reason is that Gtk requires compilation, and not everyone has a compiler installed; with this design, they won't even be able to use the package with ProfileViewSVG because it will error when they issue `using ProfileView`.

If you changed this because you couldn't get it working, what was the error?

Reply:
I'm certain there's a better solution, but I neither managed to understand the subtleties of how the code used to work nor more exactly why it was failing. Without the last commit the error is:
```
ERROR: LoadError: LoadError: UndefVarError: ProfileViewGtk not defined
Stacktrace:
 [1] #view#23(::Bool, ::Nothing, ::Bool, ::Int64, ::Bool, ::Array{Any,1}, ::Function, ::Array{UInt64,1}) at /tmp/ProfileView.jl/src/ProfileView.jl:54
 [2] view at /tmp/ProfileView.jl/src/ProfileView.jl:54 [inlined] (repeats 2 times)
 [3] top-level scope at none:0
 [4] include at ./boot.jl:317 [inlined]
 [5] include_relative(::Module, ::String) at ./loading.jl:1038
 [6] include(::Module, ::String) at ./sysimg.jl:29
 [7] include(::String) at ./client.jl:398
 [8] top-level scope at none:0
 [9] include at ./boot.jl:317 [inlined]
 [10] include_relative(::Module, ::String) at ./loading.jl:1038
 [11] include(::Module, ::String) at ./sysimg.jl:29
 [12] include(::String) at ./client.jl:398
 [13] top-level scope at none:0
in expression starting at /tmp/ProfileView.jl/test/test.jl:40
in expression starting at /tmp/ProfileView.jl/test/runtests.jl:6
```